src/index.html

```diff
@@ -6,8 +6,40 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="product-card">
+      <a href="#">
+        <img
+          class="product-card__image"
+          src="images/imac.jpeg"
+          alt="123"
```

Comment:
Bad naming for `alt` attribute. read in checklist.

Reply:
Still not fixed from the previous review
<img width="835" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/e46be43c-f56a-45b0-ab90-edcec3196f7e">